--- /opt/conda/envs/fastai/lib/python3.7/site-packages/torch/nn/modules/dropout.py
+++ /opt/conda/envs/fastai/lib/python3.7/site-packages/torch/nn/modules/dropout.py
@@ -1,9 +1,8 @@
 class Dropout(_DropoutNd):
-    r"""During training, randomly zeroes some of the elements of the input tensor with probability :attr:`p`.
-
-    The zeroed elements are chosen independently for each forward call and are sampled from a Bernoulli distribution.
-
-    Each channel will be zeroed out independently on every forward call.
+    r"""During training, randomly zeroes some of the elements of the input
+    tensor with probability :attr:`p` using samples from a Bernoulli
+    distribution. Each channel will be zeroed out independently on every forward
+    call.
 
     This has proven to be an effective technique for regularization and
     preventing the co-adaptation of neurons as described in the paper
@@ -32,6 +31,6 @@
         detectors: https://arxiv.org/abs/1207.0580
     """
 
-    def forward(self, input: Tensor) -> Tensor:
+    def forward(self, input):
         return F.dropout(input, self.p, self.training, self.inplace)
 